.yarnrc.yml

```diff
@@ -1,7 +1,12 @@
 nodeLinker: node-modules
 
+npmScopes: 
+  kimafinance:
+    npmRegistryServer: "https://registry.npmjs.org/"
+    npmAuthToken: "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"
+
```

Comment:
@evergreen9001 you probably want to remove this and revoke token

Reply:
@sirpy will do once we made the widget public npm.

Reply:
@sirpy This is done.